MWPhotoBrowser.podspec

```diff
@@ -39,6 +39,6 @@ Pod::Spec.new do |s|
 
   s.dependency 'MBProgressHUD', '~> 0.9'
   s.dependency 'DACircularProgress', '~> 2.3'
-  s.dependency 'SDWebImage', '~> 4.0'
+  s.dependency 'SDWebImage', '~> 4.3.3'
```

Comment:
I was not sure if I should leave it like it was or to specify the latest version?

Reply:
I would *lock* to s spacific version

Reply:
i would NOT lock, as this is a transitive dependency, and pod installation would fail if another pod also depended on `SDWebImage`

Reply:
I forgot that it is not our direct dependency.

Reply:
maybe `~> 4.3` so we receive updates to 4.4, 4.5 as well?